docs/src/installation.md

```diff
@@ -29,8 +50,6 @@ Pkg.build("RCall")
 
 When `R_HOME` is set to `"*"`, RCall.jl will automatically install R for you using [Conda](https://github.com/JuliaPy/Conda.jl).
 
-Should you experience problems with any of these methods, please [open an issue](https://github.com/JuliaStats/RCall.jl/issues/new).
```

Comment:
I think this blurb should stay in.

Reply:
I moved it to the top level title "Customizing the R installation" so it's clear it's for all methods.